tutorials/burn-a-dvd-on-macos/burn-a-dvd-on-macos.md

```diff
@@ -6,39 +6,50 @@ tags: tutorial,installation,burn,dvd,macOS,ubuntu,desktop
 difficulty: 2
 status: Published
 published: 2017-05-31
+author: Canonical Web Team <webteam@canonical.com>
 
 ---
 
 # How to burn a DVD on macOS
 
-## Getting started
+## Overview
 
-Before you begin, you need to [download](https://www.ubuntu.com/download) Ubuntu and, if you want, [verify the download](/tutorial/tutorial-how-to-verify-ubuntu) .
+To create an Ubuntu installation DVD on macOS, you will need the following:
 
-The easiest way to burn an ISO, the file you need to install Ubuntu from a DVD, is by right clicking on the file in the Finder. If you are using an older version of macOS, you might need to use the Disk Utility, or on even older versions of macOS, you might need to use use Disk Copy.
+* An Ubuntu ISO you can [download here](https://www.ubuntu.com/download)
+* A blank DVD
+
+positive
+: **Ensuring the authenticity of an Ubuntu ISO**
```

Comment:
How about a more bold and direct statement then:

The download can be [verified] to ensure...